### PR TITLE
allow strict option for bodyParser

### PR DESCRIPTION
**Pull request by dereklin**
https://github.com/webpro/dyson/issues/102

### CODE REVIEW

File: lib/dyson.js [hunk 1]
```diff
@@ -49,7 +49,13 @@ const setConfig = config => (req, res, next) => {
 };
 
 const installMiddleware = (app, options) => {
-  const bodyParserOptions = options.bodyParserJsonLimit ? { limit: options.bodyParserJsonLimit } : {};
+  const bodyParserOptions = {};
+  if (options.bodyParserJsonLimit) {
+    bodyParserOptions.limit = options.bodyParserJsonLimit;
+  }
+  if (options.strict !== undefined) {
+    bodyParserOptions.strict = options.strict;
```
Comment (webpro) [inline lib/dyson.js]: Please rename `strict` to `bodyParserJsonStrict`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dyson",
-  "version": "2.0.5",
+  "version": "2.0.6",
```
Comment (webpro) [inline package.json]: Please don't update the `version`, I do releases with [release-it](https://github.com/release-it/release-it).

File: lib/dyson.js (hunk 1 above)
Comment (dereklin) [reply lib/dyson.js]: Will do.

File: package.json (hunk 2 above)
Comment (dereklin) [reply package.json]: Got it.

**Comment by webpro** [comment]
Thanks @dereklin, this has just been released in [v3.0.0](https://github.com/webpro/dyson/releases/tag/3.0.0)

**Pull request merged**